tests/NuGet.Services.Validation.Orchestrator.Tests/ValidationSetProviderFacts.cs

```diff
@@ -490,9 +526,15 @@ public async Task DoesNotCreateValidationsWhenShouldStartFalse()
         public async Task DoesNotEmitTelemetryIfMultipleValidationSetsExist()
         {
             const string validation1 = "validation1";
-            Configuration.Validations = new List<ValidationConfigurationItem>
+            Configuration.ValidationSteps = new Dictionary<string, List<ValidationConfigurationItem>>
             {
-                new ValidationConfigurationItem(){ Name = validation1, TrackAfter = TimeSpan.FromDays(1), RequiredValidations = new List<string>{ } }
+                {
+                    ValidationStepsContentType.NuGet,
+                    new List<ValidationConfigurationItem>
+                    {
+                        new ValidationConfigurationItem(){ Name = validation1, TrackAfter = TimeSpan.FromDays(1), RequiredValidations = new List<string>{ } }
```

Comment:
Nit: 

```suggestion
                        new ValidationConfigurationItem { Name = validation1, TrackAfter = TimeSpan.FromDays(1), RequiredValidations = new List<string>() }
```

Alternatively consisder:

```suggestion
                        new ValidationConfigurationItem
                        {
                            Name = validation1,
                            TrackAfter = TimeSpan.FromDays(1),
                            RequiredValidations = new List<string>()
                        }
```